ps-modules/Util/Util.psm1

```diff
@@ -63,9 +63,48 @@ function Write-ReleaseInfoJson {
     $releaseInfo | ConvertTo-Json | Set-Content -Path $PathToJsonFile
 }
 
-function Get-PackageNameOnly {
-   param (
-      [string] $PackageNameAndFeatures
-   )
-   return $PackageAndFeatures -replace '\[.*$', ''
-}
\ No newline at end of file
+function Get-IsOnWindowsOS {
```

Comment:
Here's another spot we can use this new function: https://github.com/TechSmith/ThirdParty-Packages-vcpkg/pull/1/files#diff-ebc5a580a121e62d1f8eeb8ce872c9e223970e428f8fb8533d8f9ea0f692b396R19

Reply:
This is now used throughout the various scripts.